zerver/openapi/zulip.yaml

```diff
@@ -3680,6 +3688,10 @@ paths:
                                       type: boolean
                                       description: |
                                         Whether [topics are required](/help/require-topics) for messages in this organization.
+                                    signup_notifications_include_referrer:
+                                      type: boolean
+                                      description: |
+                                        Whether the referrer's name has to be mentioned in the signup notification in the core-team stream.
```

Comment:
"Whether new user signup notifications contain the referrer's name." would be a clearer description here and in the UI.